examples/python/monitoring/10_comprehensive_dashboard.py

```
@@ -0,0 +1,432 @@
+        error_rate = summary['total_errors'] / summary['total_function_calls'] * 100
+        print(f"  Error Rate: {error_rate:.2f}%")
+    else:
+        print(f"  Error Rate: 0.00%")
+    
+    # Top Performers and Issues
+    print(f"\n🏆 TOP PERFORMERS")
+    perf_data = dashboard_data['performance_data']
+    
+    print("  Fastest Functions:")
+    fastest_functions = sorted(perf_data['slowest_functions'], key=lambda x: x[1])[:3]
+    for name, time in fastest_functions:
```

Ruff (0.12.2)

Import `time` from line 13 shadowed by loop variable

(F402)